GA/HeaderFiles/DataWrite.h

```diff
@@ -22,9 +22,11 @@ void DataWrite(std::vector<std::vector<std::vector<float> > >& dna_output,
   // Write the header
   ofstream generationDNA;
   generationDNA.open("generationDNA.csv");
-  generationDNA << "Hybrid of Roulette and Tournament -- Thanks to Cal Poly / Jordan Potter" << "\n";
+  generationDNA << "Hybrid of Roulette and Tournament -- "
+                   "Thanks to Cal Poly / Jordan Potter" << "\n";
   generationDNA << "Author was David Liu" << "\n";
-  generationDNA << "Notable contributors: Julie Rolla, Hannah Hasan, and Adam Blenk" << "\n";
+  generationDNA << "Notable contributors: Julie Rolla, Hannah Hasan, "
+                   "and Adam Blenk" << "\n";
```

Comment:
PR should generally focus on a single feature/issue and shouldn't include extraneous changes (unless it's a minor change and you are already touching that part of the code).  If, for example, this PR gets rejected or reverted, this unrelated change will also get lost.

Reply:
Hi Ezio,
Sorry I didn't know it's going to send notification to you each time I make an edit.
I have reverted the main branch to what its original state (before any HPol commits) and then squashed the HPol branch into one commit.
I'll make one last edit on HPol following the comments above and then stop editing HPol until we are ready to merge HPol back to main. 
In the meantime I'll just make a separate branch to poke at.

Reply:
Don't worry about the notifications, I can tweak the settings to be notified about what I want (and I get plenty of notifications every day anyway :).

The usual workflow is:
1. create a local branch with some commits
2. push the branch on GitHub (`git push origin your-branch`) and create a PR (it gives you a link to create it after the push)
3. wait for reviews and comments
4. add more commits to the local branch to address the reviews/comments
5. push them to GitHub (`git push origin your-branch`, without forcing/squashing/rebasing) -- the PR will update automatically to include the new commit
6. repeat steps 3-5 until the PR is approved
7. click on the squash merge button on GitHub to merge the PR on `main`

Reply:
Understood. Thank you!